spring-batch/src/main/java/org/baeldung/conditionalflow/step/NotifierTasklet.java

```diff
@@ -0,0 +1,14 @@
+package org.baeldung.conditionalflow.step;
+
+import org.springframework.batch.core.StepContribution;
+import org.springframework.batch.core.scope.context.ChunkContext;
+import org.springframework.batch.core.step.tasklet.Tasklet;
+import org.springframework.batch.repeat.RepeatStatus;
+
+public class NotifierTasklet implements Tasklet {
```

Comment:
Formatting again.
Please also use a logger as mentioned previously instead of system statements.

Reply:
If we use a logger instead of System.out we lose the output of the step when logging to file - IMO System.out is appropriate here as it's not logging, it's demonstrating where we would do stuff in the real world.